doc/sphinx-guides/source/admin/metadataexport.rst

```diff
@@ -28,7 +28,7 @@ Note, that creating, modifying, or re-exporting an OAI set will also attempt to
 Export Failures
 ---------------
 
-An export batch job, whether started via the API, or by the application timer, will leave a detailed log in your configured logs directory. This is the same location where your main Glassfish server.log is found. The name of the log file is ``export_[timestamp].log`` - for example, *export_2016-08-23T03-35-23.log*. The log will contain the numbers of datasets processed successfully and those for which metadata export failed, with some information on the failures detected. Please attach this log file if you need to contact Dataverse support about metadata export problems.
+An export batch job, whether started via the API, or by the application timer, will leave a detailed log in your configured logs directory. This is the same location where your main Payara server.log is found. The name of the log file is ``export_[timestamp].log`` - for example, *export_2016-08-23T03-35-23.log*. The log will contain the numbers of datasets processed successfully and those for which metadata export failed, with some information on the failures detected. Please attach this log file if you need to contact Dataverse support about metadata export problems.
```

Comment:
Can probably change this one (i.e. no matter what server we ever use, all logs should be in same place)

Reply:
Fixed in db70a247c.